### ISSUE TITLE
cx side effects needed

### ISSUE THREAD

**Issue by steady-clackin** [skycoin/cx#7]
Thinking ahead to blockchain apps, one use case might be a messaging system that uses the blockchain for the shared message room data. However, this mechanism will be too slow for realtime communications. When say a `sendmsg` "command" is sent to the blockchain somehow triggering the sendmsg cx code, it would be nice for the cx code to have an idempotent side effect of pushing data to some cxo feed for immediate consumption by clients. Some time later the feed msgs could be reconciled with the blockchain state. This would allow for a fast "eventually consensus correct" messaging system.

**Comment by drew-512** [comment]
Yeah, I love this insight.   I like how this essentially is a shoutout to the principle that commit latency can be lowered if/when there's protocol assurance that consensus probability can be estimated at any given period of time.
